scripts/dev/create_dca_policy/main.go

```diff
@@ -78,14 +78,13 @@ func main() {
 
 	policyId := uuid.New().String()
 	policy := types.PluginPolicy{
-		ID:            policyId,
-		PublicKey:     key,
-		PluginID:      dca.PluginType,
-		PluginVersion: dca.PluginVersion,
-		PolicyVersion: dca.PolicyVersion,
-		PluginType:    dca.PluginType,
-		Active:        true,
-		Signature:     "0x0000000000000000000000000000000000000000000000000000000000000000",
+		ID:             policyId,
+		PublicKeyEcdsa: key,
+		PluginVersion:  dca.PluginVersion,
+		PolicyVersion:  dca.PolicyVersion,
+		PluginType:     dca.PluginType,
+		Active:         true,
+		Signature:      "0x0000000000000000000000000000000000000000000000000000000000000000",
 	}
 
 	payrollPolicy := dca.Policy{
```

Comment:
rename this

Reply:
https://github.com/vultisig/vultiserver-plugin/pull/90/commits/3b9f76cc4bb9d57d8e3a82e2f05344e406ba21b6